docs/src/inputs/input/devSandbox/devSandbox.jsx

```diff
@@ -0,0 +1,565 @@
+import {
+    Typography,
+} from 'react-cm-ui';
+import {
+    camelCase,
+} from 'lodash';
+import React from 'react';
+import Example from '../../../global/example';
+import ExampleInputAutocomplete from '../examples/exampleInputAutocomplete';
+import ExampleInputAutoFocus from '../examples/exampleInputAutoFocus';
+import ExampleInputDisabled from '../examples/exampleInputDisabled';
+import ExampleInputError from '../examples/exampleInputError';
+import ExampleInputErrorWithText from '../examples/exampleInputErrorWithText';
+import ExampleInputFluid from '../examples/exampleInputFluid';
+import ExampleInputIcon from '../examples/exampleInputIcon';
+import ExampleInputIconCustom from '../examples/exampleInputIconCustom';
+import ExampleInputInverse from '../examples/exampleInputInverse';
+import ExampleInputLabel from '../examples/exampleInputLabel';
+import ExampleInputLabelPosition from '../examples/exampleInputLabelPosition';
+import ExampleInputLoading from '../examples/exampleInputLoading';
+import ExampleInputMaxMin from '../examples/exampleInputMaxMin';
+import ExampleInputMaxLength from '../examples/exampleInputMaxLength';
+import ExampleInputMinLength from '../examples/exampleInputMinLength';
+import ExampleInputOnBlur from '../examples/exampleInputOnBlur';
+import ExampleInputOnChange from '../examples/exampleInputOnChange';
+import ExampleInputOnClick from '../examples/exampleInputOnClick';
+import ExampleInputOnFocus from '../examples/exampleInputOnFocus';
+import ExampleInputOnKeyDown from '../examples/exampleInputOnKeyDown';
+import ExampleInputPlaceholder from '../examples/exampleInputPlaceholder';
+import ExampleInputRequired from '../examples/exampleInputRequired';
+import ExampleInputType from '../examples/exampleInputType';
+import ExampleInputMask from '../examples/exampleInputMask';
+import Heading from '../../../global/heading';
+import Main from '../../../global/main';
+import MarkdownContainer from '../../../global/markdownContainer';
+/* eslint-disable import/no-named-default, import/extensions */
+import { default as componentDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/inputs/input/input';
+/* eslint-enable import/no-named-default, import/extensions */
+
+function InputDevSandbox() {
+    const {
+        description: componentDescription,
+        displayName: componentName,
+    } = componentDoc;
+
+    return (
+        <Main page={camelCase(componentName)}>
+            <Main.Content>
+                <MarkdownContainer>
+                    <Typography
+                        className="description"
+                        variant="body1"
+                    >
+                        {componentDescription}
+                    </Typography>
+
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Auto Complete
+                    </Heading>
```

Comment:
`anchorLink` should be something like `"auto-complete"`

Reply:
Need to fix for every example on it. Doing it now.

Reply:
Thank you sir! ;-)

Reply:
Fixed

Reply:
Looks good.  Resolving.